brainrender/scene.py

```diff
@@ -87,7 +87,7 @@ def __init__(self,
 		if atlas is None:
 			self.atlas = ABA(base_dir=base_dir,  **atlas_kwargs, **kwargs)
 		else:
-			self.atlas = atlas(base_dir=base_dir, **atlas_kwargs, **kwargs)
```

Comment:
Scene get's passed the class and creates an instance of the atlas. 
This saves a step as it avoids having to do

```
atlas = Atlas()
scene = Scene(atlas)
```

This makes sense because in the context of brainrender you'd never use `atlas` by itself. We can add the possibility of passing the instance of an atlas class if you think it'd be useful

Reply:
For the current state of brainglobe I think it will be cleaner to leave instantiation outside the Scene class, since we instantiate atlases passing location if different than default. But this schema can still be changed depending on what you guys think!

Reply:
but that;s why scene has an atlas_kwargs argument so that you can pass stuff directly to the atlas class. But you I can add the option to pass an isntance already.

Reply:
In my opinion at that point it's cleaner to instantiate it outside, it's one line more but you clearly separate the arguments for the different classes